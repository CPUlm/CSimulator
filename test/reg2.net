INPUT 
OUTPUT a, b, c
VAR
  a, b, c
IN
a = REG b
b = NOT a
c = b
