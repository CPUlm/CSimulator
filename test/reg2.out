Step 1: 
=> a = 0
=> b = 1
=> c = 1
Step 2: 
=> a = 1
=> b = 0
=> c = 0
Step 3: 
=> a = 0
=> b = 1
=> c = 1
Step 4: 
=> a = 1
=> b = 0
=> c = 0
